src/nucleus/value.mli

```diff
@@ -155,14 +159,21 @@ val lookup_signature : loc:Location.t -> Name.ident -> Tt.sig_def comp
 (** Find a signature with the given labels (in this exact order) *)
 val find_signature : loc:Location.t -> Name.label list -> (Name.signature * Tt.sig_def) comp
 
+(** Lookup a dynamic variable by name. *)
+val get_dynamic : Name.ident -> env -> dyn option
+
 (** Lookup abstracting variables. *)
 val lookup_abstracting : value list comp
 
 (** Lookup a free variable by its de Bruijn index *)
-val lookup_bound : loc:Location.t -> Syntax.bound -> value comp
+val lookup_bound : loc:Location.t -> int -> value comp
+
+val lookup_dynamic : dyn -> value comp
 
 (** For matching *)
-val get_bound : loc:Location.t -> Syntax.bound -> env -> value
+val get_bound : loc:Location.t -> int -> env -> value
+
+val get_dyn : dyn -> env -> value
```

Comment:
Please comment on this one, it's crypitc to have both `lookup_dynamic` and `lookup_dyn`. Actually, the same holds for `lookup_bound` and `get_bound`. Is there some sort of a scheme to the naming rules we're using here? Consistent naming is a good thing.